### PR TITLE
Setup/database

### LINKED ISSUE #1: Add Gems
Any gems added to the project must be approved by an instructor

- [x] Pre-approved gems are capybara, pry, faker, factory_bot_rails, orderly, simplecov, shoulda-matchers, launchy
- [ ] add rubocop file and gem

### LINKED ISSUE #2: Setup DB
- [x] Setup
- [x]  Fork this repository
- [x] Clone your fork
- [x]  From the command line, install gems and set up your DB:
- [x]  bundle
- [x] rails db:create
- [x]  Run the test suite with bundle exec rspec.
- [x] Run your development server with rails s to see the app in action.
- [x] set up rake tasks
- [x] add database tables for all 
- [x] add relationships to models
- [x] add tests for relationships to spec/models
- [x] add model.destroy all to seed.rb file

### PR DESCRIPTION
**Pull request by AlexKiselich**
## Checklist before requesting a review
- [ ] Verified Render deploy is functional
- [x] Pulled/merged main 
- [x] Tests for the changes have been added into spec files
- [x] Branch name follows our guidelines 
- [x] All tests running clean
- [ ] Ran this in Rails S  - features are working properly
- [x] Test coverage is at 100% for models and features

## What changes or new behavior does this PR introduce (Bug fix, feature, update, etc)?
  Added Database and setup 
## Bugs, Issues, etc:
 - [ ] check if there are known bugs or issue

## Other helpful info:

### CODE REVIEW

File: app/models/invoice.rb
Comment (kbergstrom78) [inline app/models/invoice.rb:1]: Does this need a relationship to transaction?

Comment (kbergstrom78) [reply app/models/invoice.rb:1]: Discussed in group - will fix and repull

File: spec/models/invoice_spec.rb
Comment (kbergstrom78) [inline spec/models/invoice_spec.rb:1]: This needs to have the association to transaction, too - discussed in group and will fix.

**Review by kbergstrom78 — COMMENTED** [review]
As discussed with group...

**Review by aplasket — APPROVED** [review]

**Review by kbergstrom78 — APPROVED** [review]
Comments resolved - looks good!

**Review by jpnelson85 — COMMENTED** [review]
looks good

**Pull request merged**